src/main/kotlin/util/SphereNodes.kt

```diff
@@ -69,7 +71,7 @@ class SphereNodes //FAILED to hook up here a 'parentNode' listener that would se
             }
         }
 /*
-		println("Drawing currently in total "+visibleNodeCount
+		logger.debug("Drawing currently in total "+visibleNodeCount
```

Comment:
thanks for fixing it even inside the commented-out code :+1:
(one never knows... )